app/services/scan_progress.py

```
@@ -230,6 +242,31 @@ def start_file_operation(
+                            "path_id": progress.path_id,
+                            "operation_id": op.operation_id,
+                            "file_name": op.file_name,
+                            "file_path": op.file_path,
+                            "operation": op.operation,
+                            "bytes_total": op.bytes_total,
+                            "bytes_transferred": op.bytes_transferred,
+                            "percent": op.percent,
+                            "current_speed": op.current_speed,
+                            "eta": op.eta,
+                        }
+                    )
```

Ruff (0.15.7)

[warning] Use `list.extend` to create a transformed list

(PERF401)